wbooks/ReactotronConfig.js

```diff
@@ -0,0 +1,5 @@
+import Reactotron from 'reactotron-react-native';
+
+Reactotron.configure().useReactNative().connect();
+
+console.tron = Reactotron;
```

Comment:
* Bad file name
* Bad folder structure
* Use TS (you can check this [file](https://docs.google.com/document/d/1i_Dq2sW6CjePw40i_84Y1TVXle7he2q6CnUekkVDAZI/edit))
* Missing export Reactotron after configuration
* Use `if (__DEV__)`  inside this file